### ISSUE TITLE
Change the mouse button

### ISSUE THREAD

**Issue by bespokecomp** [Mikhail22/drag-scroll--autohotkey#4]
Hi,
Really appreciate this script. 
Could you help me?

Which parameters should I change If I want to use Control & Left Mouse Button held down and dragged as scroll up/down?

I need the right mouse button to remain as its default setting.

Many thanks

**Comment by Mikhail22** [comment]
@bespokecomp yes you could try that, here is an example code, it is for the version v04 (check the repo if you use older version). Replace the last 2 hotkeys with following:  

```
^LButton::
	running := 1
	dyTotal := 0
	mousegetpos, mxLast, myLast
return

^LButton up::
	running := 0
	scrollsTotal := 0
	send, {Lbutton up} 
return
```

This is not a great idea though, it may conflict with some apps because Ctrl+wheel may leak to applications and some use it as zoom in/out.

**Comment by bespokecomp** [comment]
You are a gentleman and a scholar!  Thank you.

I am using with a pen tablet and this works brilliantly for zooming in fusion 360!
